backend/mempool-config.sample.json

```diff
@@ -15,9 +15,10 @@
     "INDEXING_BLOCKS_AMOUNT": 11000,
     "PRICE_FEED_UPDATE_INTERVAL": 600,
     "USE_SECOND_NODE_FOR_MINFEE": false,
-    "EXTERNAL_ASSETS": [
-      "https://raw.githubusercontent.com/mempool/mining-pools/master/pools.json"
-    ],
+    "EXTERNAL_ASSETS": [],
+    "EXTERNAL_MAX_RETRY": 10,
+    "EXTERNAL_RETRY_INTERVAL": 60,
```

Comment:
@ayanamidev this should be in milliseconds to match `POLL_RATE_MS` set just a few lines above

Reply:
No problem

Reply:
@knorrium Checking this again looks like there is a similar config called `PRICE_FEED_UPDATE_INTERVAL` which is a config defined with seconds not ms. If the config key doesn't include the term `MS`, wouldn't it be good to leave this value as is?